src/Carousel.tsx

```diff
@@ -180,10 +180,10 @@ const defaultProps = {
   wrap: true,
   touch: true,
 
-  prevIcon: <span aria-hidden="true" className="carousel-control-prev-icon" />,
+  prevIcon: <span aria-label="previous slide" aria-hidden="true" className="carousel-control-prev-icon" />,
```

Comment:
The carousel also renders `<span class="sr-only">Previous</span>` like in the upstream Bootstrap examples.  Are these not working properly for screen readers?

Reply:
Seems nothing wrong with that! However, adding `aria-label` was necessary because I've got error on one project in lighthouse test.